lib/instrumentation/aws-sdk/util.js

```diff
@@ -32,7 +32,17 @@ function setDynamoParameters(endpoint, params) {
   })
 }
 
+function setRedshiftParameters(endpoint, params) {
```

Comment:
I would just keep this in the redshift instrumentation. The only reason dynamo functions are in here is to facilitate it being used in multiple places